secure_message/repository/modifier.py

```diff
@@ -3,6 +3,9 @@
 from flask import jsonify
 from structlog import wrap_logger
 from werkzeug.exceptions import InternalServerError
+
+from secure_message.common import events
```

Comment:
from secure_message.common import events is not needed as we are importing Events ?